pennylane/ops/qubit.py

```diff
@@ -1574,8 +1600,18 @@ def _matrix(cls, *params):
             [
                 [1, 0, 0, 0],
                 [0, 1, 0, 0],
-                [0, 0, cmath.exp(-0.5j * (phi + omega)) * c, -cmath.exp(0.5j * (phi - omega)) * s],
-                [0, 0, cmath.exp(-0.5j * (phi - omega)) * s, cmath.exp(0.5j * (phi + omega)) * c],
+                [
```

Comment:
Just as a marker, it would be nice to have these changes reverted. That will help with looking through the real changes of the PR (even later down the line).